sensor/kubernetes/listener/resources/networkpolicy_store.go

```diff
@@ -199,6 +199,7 @@ func (n *networkPolicyStoreImpl) Find(namespace string, podLabels map[string]str
 func (n *networkPolicyStoreImpl) OnNamespaceDeleted(namespace string) {
 	n.lock.Lock()
 	defer n.lock.Unlock()
+	defer n.updateStateMetric()
```

Comment:
I realized that when we clear a namespace completely the metrics will show one remaining object in the store. This is because we delete the top level object before sending the metrics update, therefore prometheus only gets updates in the remaining namespaces. It's a minor issue but something to take into consideration.